Randomizer.cpp

```diff
@@ -844,9 +844,9 @@ namespace Randomizer {
         */
         if (options.race) {
             std::cout << "Race mode enabled, altering actual seed.\n";
-            Random::RandomInit(seed);
-            Random::RandomInit(seed);
-            Random::RandomInit(seed);
+            Random::RandomInteger(100);
```

Comment:
You sure about the 100 here? Shouldn't you use `seed` here?

Reply:
It actually doesn't matter, we just need to do something that advances the RNG. Maybe I should use a different function entirely.

Reply:
Hey there actually is a function to do exactly this! Thanks for the reviews, it is really helping with code quality here.